### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by athar05**
This is the landing page of the app. 

It contains the links to different pages of the app and is fully mobile responsive.

### CODE REVIEW

File: src/components/LandingBody.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  return (
+    <section className="landing">
+    <div>
+      <div className="landing-inner">
+        <h1 className="x-large">Entrepreneur Connector</h1>
+        <p className='lead'>
+          A platform to connect with other entrepreneurs, VCs and peers.
+        </p>
+        <Stack direction='row' spacing={2}>
+          <Link to="/login" style={{textDecoration: 'none'}}><Button variant='contained' className="buttons">Login</Button></Link>
```
Comment (vivekgudipudi) [inline src/components/LandingBody.js]: try to create a separate class for this styling as it is used many times

**Review by vivekgudipudi — APPROVED** [review]
Good use of CSS variables and nice structure.

File: src/pages/LandingPage.js [hunk 2]
```diff
@@ -1,9 +1,12 @@
-import React, {Fragment} from 'react'
+import React, {Fragment} from 'react';
+import Navbar from '../components/Navbar';
+import LandingBody from '../components/LandingBody';
```
Comment (chanchal16) [inline src/pages/LandingPage.js:3]: You can use barrel exports here.

File: src/components/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+            <IconButton size="large" edge="start" color="inherit" aria-label='logo'> 
+                <ConnectWithoutContactIcon/ >
+            </IconButton>
+            <Typography variant='h6' component='div' sx={{flexGrow: 1}} className="brand-logo">
+                ENTREPRENEUR CONNECT
+            </Typography>
+            <Stack direction='row' spacing={2}>
+            <Link to='/login' style={{textDecoration: 'none'}}><Button style={{color: 'white'}}>Login</Button></Link>
+            <Link to='/register' style={{textDecoration: 'none'}}><Button style={{color: 'white'}}>Register</Button></Link>
+            </Stack>
```
Comment (chanchal16) [inline src/components/Navbar.js:19]: Create a separate class as its used multiple times and try to avoid inline styling

**Review by chanchal16 — APPROVED** [review]
Left some sanity comments other than that looks good

**Review by sohan9819 — APPROVED** [review]
Other than the changes suggested , rest looks good 👍

**Review by DivyanshSareen — APPROVED** [review]
Good use of comments in style sheets. Good to merge

File: src/components/LandingBody.js (hunk 1 above)
Comment (athar05) [reply src/components/LandingBody.js]: Acknowledged. Thanks.

File: src/pages/LandingPage.js (hunk 2 above)
Comment (athar05) [reply src/pages/LandingPage.js:3]: Acknowledged. Thanks.

File: src/components/Navbar.js (hunk 3 above)
Comment (athar05) [reply src/components/Navbar.js:19]: Acknowledged. Thanks.

**Pull request merged**